src/ucs/sys/topo.c

```diff
@@ -22,7 +22,8 @@
 
 
 #define UCS_TOPO_MAX_SYS_DEVICES     256
-#define UCS_TOPO_SYSFS_PCI_PREFIX    "/sys/class/pci_bus"
+#define UCS_TOPO_SYSFS_PCI_PREFIX    "/sys/bus/pci/devices/"
+#define UCS_TOPO_SYSFS_DEVICES_ROOT  "/sys/devices"
```

Comment:
add trailing "/" to make it clear that it is a directory

Reply:
Iit will make ucs_topo_is_sys_root() more complicated since need to compare without the trailing "/", WDYT?

Reply:
> Iit will make ucs_topo_is_sys_root() more complicated since need to compare without the trailing "/", WDYT?

i see, thanks